src/query.c

```diff
@@ -1244,21 +1244,34 @@ static sds QueryNode_DumpSds(sds s, const IndexSpec *spec, const QueryNode *qs,
       break;
     case QN_IDS:
 
-      s = sdscat(s, "IDS { ");
+      s = sdscat(s, "IDS {");
       for (int i = 0; i < qs->fn.len; i++) {
         s = sdscatprintf(s, "%llu,", (unsigned long long)qs->fn.ids[i]);
       }
       break;
     case QN_VECTOR:
-      // TODO: add vector query params
-      s = sdscat(s, "VECTOR { ");
-      VecSimQueryResult_Iterator *iter = VecSimQueryResult_List_GetIterator(qs->vn.vq->results);
-      VecSimQueryResult *res = VecSimQueryResult_IteratorNext(iter);
-      while (res) {
-        s = sdscatprintf(s, "[%lu,%lf]", VecSimQueryResult_GetId(res), VecSimQueryResult_GetScore(res));
-        res = VecSimQueryResult_IteratorNext(iter);
+      s = sdscat(s, "VECTOR {");
+      if (QueryNode_NumChildren(qs) > 0) {
+        s = sdscat(s, "\n");
+        s = QueryNode_DumpChildren(s, spec, qs, depth + 1);
+        s = doPad(s, depth);
+        s = sdscat(s, "} => {");
+      }
+      switch (qs->vn.vq->type) {
+        case VECSIM_QT_TOPK: {
+          s = sdscatprintf(s, "TOP %zu vectors similar to `", qs->vn.vq->topk.k);
```

Comment:
The query is using `TOP_K`, why not use same token instead of `TOP`, or mention the `K`?
e.g.,
`"TOP K=%zu vectors similar to `"`

Reply:
And need test